bin/resources/GameIndex.yaml

```diff
@@ -29653,7 +29657,9 @@ SLPM-55244:
     halfPixelOffset: 2 # Fixes blurriness.
     getSkipCount: "GSC_NFSUndercover"
 SLPM-55245:
-  name: "Himawari - Pebble in the Sky"
+  name: ひまわり - Pebble in the Sky -
+  name-sort: ひまわり - Pebble in the Sky -
+  name-en: "Himawari - Pebble in the Sky"
```

Comment:
No spaces around the dashes for the Japanese title: http://never-island.com/himawari/product.html